portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -13,8 +13,10 @@
 // limitations under the License.
 
 package com.google.sps.servlets;
```

Comment:
Adding this comment so we don't forget to update the format using Google Java Format